src/test/java/uk/gov/hmcts/reform/divorce/orchestration/workflows/SendCoRespondSubmissionNotificationWorkflowTest.java

```diff
@@ -65,9 +69,17 @@
     @Mock
     private TaskCommons taskCommons;
 
+    @Mock
+    private CcdUtil ccdUtil;
+
     @InjectMocks
     private SendCoRespondSubmissionNotificationWorkflow classToTest;
 
+    @Before
+    public void before() throws TaskException {
+        when(ccdUtil.getFormattedDueDate(any(), any())).thenReturn(TEST_EXPECTED_DUE_DATE_FORMATTED);
```

Comment:
Move this mock to each place needed and to make sure the method is using the valid field use
```suggestion
        when(ccdUtil.getFormattedDueDate(ccdCallbackRequest.getCaseDetails().getCaseData(), CO_RESPONDENT_DUE_DATE)).thenReturn(TEST_EXPECTED_DUE_DATE_FORMATTED);
```